CMakeLists.txt

```diff
@@ -112,13 +110,12 @@ endif()
 # See https://github.com/NOAA-EMC/NCEPLIBS-nemsio/pull/22
 target_link_libraries(nemsio::nemsio INTERFACE w3emc::w3emc_d bacio::bacio_4)
 
-if(CCPP)
-  find_package(Python 3 QUIET COMPONENTS Interpreter)
-  if (NOT Python_Interpreter_FOUND)
-    find_package(Python 2.7 REQUIRED COMPONENTS Interpreter)
-  endif()
-  message("Found Python: ${Python_EXECUTABLE}")
+# Configure Python
+find_package(Python 3 QUIET COMPONENTS Interpreter)
+if (NOT Python_Interpreter_FOUND)
+  find_package(Python 2.7 REQUIRED COMPONENTS Interpreter)
```

Comment:
Should we keep using 2.7?  When do we deprecate it?

Reply:
2.7 is only used if 3 isn't found. I think all we need to do is to check that each of the systems we are using, including the generic platforms and the dinosaurs (wcoss_cray), has Python 3 and test if not used already. Then remove this logic in a future PR.

Reply:
I created an issue for this so that we don't forget.